templates/android/template/CMakeLists.txt

```diff
@@ -12,4 +12,7 @@ include(${CMAKE_CURRENT_LIST_DIR}/../common/CMakeLists.txt)
 
 cc_android_before_target(${CC_LIB_NAME})
 add_library(${CC_LIB_NAME} SHARED ${CC_ALL_SOURCES})
+
+set_target_properties(${CC_LIB_NAME} PROPERTIES LINK_FLAGS " -Wl,--gc-sections")
+
```

Comment:
These changes should be merge into `templates/cmake/android.cmake` without modifying templates